.github/workflows/pebble.yml

```diff
@@ -2,7 +2,7 @@ name: Pebble
 
 on:
   push:
-    branches: ['main']
+    branches: ['main', '*_ci']
```

Comment:
Very nitty, but can we make it `-` instead of `_`?

Reply:
Changed.